src/main/java/org/opentripplanner/routing/core/vehicle_sharing/Provider.java

```diff
@@ -1,20 +1,20 @@
 package org.opentripplanner.routing.core.vehicle_sharing;
 
 public class Provider {
-    private final int id;
+    private final int providerId;
```

Comment:
Skąd te zmiany właściwie? Wiadomo, że pole `id` w klasie `Provider` odnosi się do id providera xD

Reply:
jest problem z deserializacją json -> VehicleDescription, bo było combo immutable classs + unwrapped zrobione nie przez unwrapped, a gettery. W ten sposób to działa.

Mogłem dać `@JsonUnwrapped(suffix="provider")` ale wtedy jest api break bo `providerId` zmienia się na `providerid`.
A jak nic nie zrobię to nie mogę tego deserializować spowrotem i testy by trzeba byłio pisać na suchym jsonie.